src/KafkaNET.Library/ZooKeeperIntegration/Listeners/ZKRebalancerListener.cs

```diff
@@ -496,34 +496,19 @@ private void AddPartitionTopicInfo(ZKGroupTopicDirs topicDirs, string partition,
             }
             var leader = leaderOpt.Value;
             var znode = topicDirs.ConsumerOffsetDir + "/" + partition;
-            var offsetString = this.zkClient.ReadData<string>(znode, true);
+
+            var offsetCommitedString = this.zkClient.ReadData<string>(znode, true);
 
             //if first time starting a consumer, set the initial offset based on the config
-            long offset = 0;
-            long offsetCommited = 0;
-            if (offsetString == null)
-            {
-                switch (config.AutoOffsetReset)
```

Comment:
i'm not sure it's such a good idea to completely remove auto offset reset and start at the beginning of the topic... this will actually break as per #39